### PR TITLE
Meta: Add a build guide to the root of the project.

### PR DESCRIPTION
**Pull request by DaimyoHub**
it would be interesting to implement a script to prepare the environment, and avoid having to compile the compiler yourself.

(Actually I don't know if the build guide is accurate, I didn't try yet.)

### CODE REVIEW

File: build_guide.md [hunk 1]
```diff
@@ -56,12 +80,14 @@ $ cd ../build_gcc
 $ make -j6 all-gcc
 $ make install-gcc
 ```
-Compile the sources with Make and install it to the previously given directory (/opt/cross-compiler).
 
 # Build the system and run it on QEMU
+Compile the system and run it in 32bits with the limine bootloader.
 ```bash
-$ make -j6 && ./run-32bits-limine.sh
+$ make -j6
```
Comment (Keyboard-Slayer) [inline build_guide.md:87]: can you change the `-j6` to `-j$(nproc)` ?

File: script/setup-deps.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
```
Comment (Keyboard-Slayer) [inline script/setup-deps.sh:1]: `./script/build-limine.sh` do that

**Review by keyboard-slayer — CHANGES_REQUESTED** [review]
There are small changes to do and it will be perfect :+1:

File: build_guide.md [hunk 3]
```diff
@@ -71,20 +71,20 @@ $ ../gcc-10.2.0/configure       \
...
+$ make -j$(nproc) all-gcc
 $ make install-gcc
 ```
 
 # Build the system and run it on QEMU
 Compile the system and run it in 32bits with the limine bootloader.
 ```bash
-$ make -j6
+$ make -j$(nproc)
 $ ./script/run-32bits-limine.sh
```
Comment (Keyboard-Slayer) [inline build_guide.md:88]: You should also call the script './script/build-limine.sh`

**Review by keyboard-slayer — CHANGES_REQUESTED** [review]

**Review by keyboard-slayer — APPROVED** [review]
LGTM :+1:

**Pull request merged**